### ISSUE TITLE
Stuck in "Starting...."

### ISSUE THREAD

**Issue by OGTWONG** [homebridge/homebridge#3274]
### Current Situation

Up To Date
v1.6.0

Synology NAS running Homebidge.

Homebridge stuck in "Starting..." and won't run, Apple HomeKit can't see accessories either.

This was working yesterday fine. Does anyone know what's going on?

### Logs

```markdown
[37m[11/29/2022, 5:51:40 PM][0m [36m[HB Supervisor][0m Creating default config.json: /volume1/homebridge/config.json
[37m[11/29/2022, 5:51:40 PM][0m [36m[HB Supervisor][0m OS: Linux 4.4.180+ x64
[37m[11/29/2022, 5:51:40 PM][0m [36m[HB Supervisor][0m Node.js v16.16.0 /volume1/@appstore/homebridge/app/bin/node
[37m[11/29/2022, 5:51:40 PM][0m [36m[HB Supervisor][0m Homebridge Path: /volume1/homebridge/node_modules/homebridge/bin/homebridge
[37m[11/29/2022, 5:51:40 PM][0m [36m[HB Supervisor][0m UI Path: /var/packages/homebridge/target/app/lib/node_modules/homebridge-config-ui-x/dist/bin/standalone.js
[0;37m[11/29/2022, 5:51:42 PM] [0m[0;36m[Homebridge UI][0m [0;33mHomebridge Config UI X v4.50.1 is listening on :: port 8581[0m
[37m[11/29/2022, 5:51:42 PM][0m [36m[HB Supervisor][0m Starting Homebridge with extra flags: -I -P /volume1/homebridge/node_modules --strict-plugin-resolution
[37m[11/29/2022, 5:51:42 PM][0m [36m[HB Supervisor][0m Started Homebridge v1.6.0 with PID: 3043
[37m[11/29/2022, 5:51:43 PM] [39mLoaded config.json with 0 accessories and 1 platforms.
[37m[11/29/2022, 5:51:43 PM] [39m---
[37m[11/29/2022, 5:51:43 PM] [39m[33mNo plugins found.[39m
[37m[11/29/2022, 5:51:43 PM] [39mLoading 1 platforms...
Setup Payload:
X-HM://0023UHM2XQKHP
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
[30m[47m                       [49m[39m
[30m[47m    ┌────────────┐     [49m[39m
[30m[47m    │ 227-70-697 │     [49m[39m
[30m[47m    └────────────┘     [49m[39m
[30m[47m                       [49m[39m
[37m[11/29/2022, 5:51:43 PM] [39mHomebridge v1.6.0 (HAP v0.11.0) (Homebridge 42DF) is running on port 51893.
Could not create mDNS advertisement. The HAP-Server won't be discoverable: Error: No such interface found
[0;37m[11/29/2022, 5:52:02 PM] [0m[0;36m[Homebridge UI][0m [0;33mAdded new user: Admin[0m
[0;37m[11/29/2022, 5:52:21 PM] [0m[0;36m[Homebridge UI][0m Running Command: /var/packages/homebridge/target/app/lib/node_modules/homebridge-config-ui-x/upgrade-install.sh 4.50.2 /var/packages/homebridge/target/app
[0;37m[11/29/2022, 5:52:30 PM] [0m[0;36m[Homebridge UI][0m Homebridge restart request received
[0;37m[11/29/2022, 5:52:31 PM] [0m[0;36m[Homebridge UI][0m Sending SIGTERM to process...
[37m[11/29/2022, 5:52:31 PM][0m [36m[HB Supervisor][0m Stopping services...
[37m[11/29/2022, 5:52:31 PM] [39mGot SIGTERM, shutting down Homebridge...
[37m[11/29/2022, 5:52:36 PM][0m [36m[HB Supervisor][0m Homebridge Process Ended. Code: 143, Signal: null
[37m[11/29/2022, 5:52:41 PM][0m [36m[HB Supervisor][0m OS: Linux 4.4.180+ x64
[37m[11/29/2022, 5:52:41 PM][0m [36m[HB Supervisor][0m Node.js v16.16.0 /volume1/@appstore/homebridge/app/bin/node
[37m[11/29/2022, 5:52:41 PM][0m [36m[HB Supervisor][0m Homebridge Path: /volume1/homebridge/node_modules/homebridge/bin/homebridge
[37m[11/29/2022, 5:52:41 PM][0m [36m[HB Supervisor][0m UI Path: /var/packages/homebridge/target/app/lib/node_modules/homebridge-config-ui-x/dist/bin/standalone.js
[0;37m[11/29/2022, 5:52:43 PM] [0m[0;36m[Homebridge UI][0m [0;33mHomebridge Config UI X v4.50.2 is listening on :: port 8581[0m
[37m[11/29/2022, 5:52:43 PM][0m [36m[HB Supervisor][0m Starting Homebridge with extra flags: -I -P /volume1/homebridge/node_modules --strict-plugin-resolution
[37m[11/29/2022, 5:52:43 PM][0m [36m[HB Supervisor][0m Started Homebridge v1.6.0 with PID: 3919
[37m[11/29/2022, 5:52:44 PM] [39mLoaded config.json with 0 accessories and 1 platforms.
[37m[11/29/2022, 5:52:44 PM] [39mLoaded 0 cached accessories from cachedAccessories.
[37m[11/29/2022, 5:52:44 PM] [39m---
[37m[11/29/2022, 5:52:44 PM] [39m[33mNo plugins found.[39m
[37m[11/29/2022, 5:52:44 PM] [39mLoading 1 platforms...
Setup Payload:
X-HM://0023UHM2XQKHP
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
[30m[47m                       [49m[39m
[30m[47m    ┌────────────┐     [49m[39m
[30m[47m    │ 227-70-697 │     [49m[39m
[30m[47m    └────────────┘     [49m[39m
[30m[47m                       [49m[39m
[37m[11/29/2022, 5:52:44 PM] [39mHomebridge v1.6.0 (HAP v0.11.0) (Homebridge 42DF) is running on port 51893.
Could not create mDNS advertisement. The HAP-Server won't be discoverable: Error: No such interface found
[37m[11/29/2022, 5:56:29 PM][0m [36m[HB Supervisor][0m Stopping services...
[37m[11/29/2022, 5:56:29 PM] [39mGot SIGTERM, shutting down Homebridge...
[37m[11/29/2022, 5:56:34 PM][0m [36m[HB Supervisor][0m Homebridge Process Ended. Code: 143, Signal: null
[37m[11/29/2022, 5:58:30 PM][0m [36m[HB Supervisor][0m OS: Linux 4.4.180+ x64
[37m[11/29/2022, 5:58:30 PM][0m [36m[HB Supervisor][0m Node.js v16.16.0 /volume1/@appstore/homebridge/app/bin/node
[37m[11/29/2022, 5:58:31 PM][0m [36m[HB Supervisor][0m Homebridge Path: /volume1/homebridge/node_modules/homebridge/bin/homebridge
[37m[11/29/2022, 5:58:31 PM][0m [36m[HB Supervisor][0m UI Path: /var/packages/homebridge/target/app/lib/node_modules/homebridge-config-ui-x/dist/bin/standalone.js
[0;37m[11/29/2022, 5:59:01 PM] [0m[0;36m[Homebridge UI][0m [0;33mHomebridge Config UI X v4.50.2 is listening on :: port 8581[0m
[37m[11/29/2022, 5:59:02 PM][0m [36m[HB Supervisor][0m Starting Homebridge with extra flags: -I -P /volume1/homebridge/node_modules --strict-plugin-resolution
[37m[11/29/2022, 5:59:02 PM][0m [36m[HB Supervisor][0m Started Homebridge v1.6.0 with PID: 18267
[37m[11/29/2022, 5:59:14 PM] [39mLoaded config.json with 0 accessories and 1 platforms.
[37m[11/29/2022, 5:59:14 PM] [39mLoaded 0 cached accessories from cachedAccessories.
[37m[11/29/2022, 5:59:14 PM] [39m---
[37m[11/29/2022, 5:59:14 PM] [39m[33mNo plugins found.[39m
[37m[11/29/2022, 5:59:14 PM] [39mLoading 1 platforms...
Setup Payload:
X-HM://0023UHM2XQKHP
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
[30m[47m                       [49m[39m
[30m[47m    ┌────────────┐     [49m[39m
[30m[47m    │ 227-70-697 │     [49m[39m
[30m[47m    └────────────┘     [49m[39m
[30m[47m                       [49m[39m
[37m[11/29/2022, 5:59:14 PM] [39mHomebridge v1.6.0 (HAP v0.11.0) (Homebridge 42DF) is running on port 51893.
Could not create mDNS advertisement. The HAP-Server won't be discoverable: Error: No such interface found
[0;37m[11/29/2022, 6:02:25 PM] [0m[0;36m[Homebridge UI][0m Homebridge restart request received
[0;37m[11/29/2022, 6:02:26 PM] [0m[0;36m[Homebridge UI][0m Sending SIGTERM to process...
[37m[11/29/2022, 6:02:26 PM][0m [36m[HB Supervisor][0m Stopping services...
[37m[11/29/2022, 6:02:26 PM] [39mGot SIGTERM, shutting down Homebridge...
[37m[11/29/2022, 6:02:31 PM][0m [36m[HB Supervisor][0m Homebridge Process Ended. Code: 143, Signal: null
[37m[11/29/2022, 6:02:36 PM][0m [36m[HB Supervisor][0m OS: Linux 4.4.180+ x64
[37m[11/29/2022, 6:02:36 PM][0m [36m[HB Supervisor][0m Node.js v16.16.0 /volume1/@appstore/homebridge/app/bin/node
[37m[11/29/2022, 6:02:36 PM][0m [36m[HB Supervisor][0m Homebridge Path: /volume1/homebridge/node_modules/homebridge/bin/homebridge
[37m[11/29/2022, 6:02:36 PM][0m [36m[HB Supervisor][0m UI Path: /var/packages/homebridge/target/app/lib/node_modules/homebridge-config-ui-x/dist/bin/standalone.js
[0;37m[11/29/2022, 6:02:38 PM] [0m[0;36m[Homebridge UI][0m [0;33mHomebridge Config UI X v4.50.2 is listening on :: port 8581[0m
[37m[11/29/2022, 6:02:38 PM][0m [36m[HB Supervisor][0m Starting Homebridge with extra flags: -I -P /volume1/homebridge/node_modules -D --strict-plugin-resolution
[37m[11/29/2022, 6:02:38 PM][0m [36m[HB Supervisor][0m Started Homebridge v1.6.0 with PID: 21369
[37m[11/29/2022, 6:02:39 PM] [39mLoaded config.json with 0 accessories and 1 platforms.
[37m[11/29/2022, 6:02:39 PM] [39mLoaded 0 cached accessories from cachedAccessories.
[37m[11/29/2022, 6:02:39 PM] [39m---
[37m[11/29/2022, 6:02:39 PM] [39m[33mNo plugins found.[39m
[37m[11/29/2022, 6:02:39 PM] [39mLoading 1 platforms...
[37m[11/29/2022, 6:02:39 PM] [39m[90mPublishing bridge accessory (name: Homebridge 42DF, publishInfo: {[39m
[90m  username: '0E:05:4E:59:42:DF',[39m
[90m  port: 51893,[39m
[90m  pincode: '***-**-***',[39m
[90m  category: 2,[39m
[90m  bind: undefined,[39m
[90m  mdns: undefined,[39m
[90m  addIdentifyingMaterial: true,[39m
[90m  advertiser: 'avahi'[39m
[90m}).[39m
Setup Payload:
X-HM://0023UHM2XQKHP
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
[30m[47m                       [49m[39m
[30m[47m    ┌────────────┐     [49m[39m
[30m[47m    │ 227-70-697 │     [49m[39m
[30m[47m    └────────────┘     [49m[39m
[30m[47m                       [49m[39m
[37m[11/29/2022, 6:02:39 PM] [39mHomebridge v1.6.0 (HAP v0.11.0) (Homebridge 42DF) is running on port 51893.
Could not create mDNS advertisement. The HAP-Server won't be discoverable: Error: No such interface found
```

### Configuration

```json
bridge	
name	"Homebridge 42DF"
username	"0E:05:4E:59:42:DF"
port	51893
pin	"227-70-697"
advertiser	"avahi"
accessories	[]
platforms	
0	
name	"Config"
port	8581
platform	"config"
```

### Environment

- OS:
- Software:
- Node:
- npm:

### Process Supervisor

other (Explain in `Additional Context`)

### Additional Context

Installed on Synology DS920+ via package centre.

**Comment by OGTWONG** [comment]
I have fixed this by changing mDNS Advertiser from "Avahi" to "Ciao" in the Homebrige settings. Would like know to know what this means though so I know.

**Comment by OxbloodUK** [comment]
I have the exact same problem on a synology 920+

**Comment by trist** [comment]
> I have fixed this by changing mDNS Advertiser from "Avahi" to "Ciao" in the Homebrige settings. Would like know to know what this means though so I know.

This also worked for me

**Comment by Supereg** [comment]
This issue with the avahi advertiser was resolved in the latest homebridge 1.6.1 update.
